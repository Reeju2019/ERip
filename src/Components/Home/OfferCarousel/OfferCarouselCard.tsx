import { Image } from 'react-bootstrap'

interface Item {
  data: string
}

const OfferCarouselCard: React.FunctionComponent<Item> = (data) => {
  return (
    <>
      <div>
        <Image src={data.data} className='d-block h-100 w-auto' alt={data.data} />
      </div>
    </>
  )
}

export default OfferCarouselCard
