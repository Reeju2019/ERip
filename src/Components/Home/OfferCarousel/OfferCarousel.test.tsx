import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import OfferCarouselCard from './OfferCarouselCard'

test('offer carousel', () => {
  render(<OfferCarouselCard />)
})

describe('pnf rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<OfferCarouselCard />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<OfferCarouselCard />)
    ReactDOM.unmountComponentAtNode(div)
  })
})
