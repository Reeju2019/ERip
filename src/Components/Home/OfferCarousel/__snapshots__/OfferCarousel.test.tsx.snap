// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pnf rendering specification SnapShot testing 1`] = `
<div>
  <img
    className="d-block h-100 w-auto"
  />
</div>
`;
